### PR TITLE
add market stats from Aleph

### PR DESCRIPTION
**Pull request by tomjohn1028**
closes: https://github.com/mithraiclabs/psyoptions-frontend/issues/775

Blocked by Aleph's `volBase24hr` is always 0

### CODE REVIEW

File: src/utils/networkInfo.ts [hunk 1]
```diff
@@ -118,7 +118,7 @@ const getDexProgramKeyByNetwork = (name: ClusterName) => {
 const getGraphQLUrlByNetwork = (name: ClusterName) => {
   switch (name) {
     case 'Mainnet':
-      return 'https://api.psyoptions.io/v1/graphql';
+      return 'https://api.serum.markets';
     case 'Devnet':
       return 'https://devnet-api.psyoptions.io/v1/graphql';
```
Comment (Taylor123) [inline src/utils/networkInfo.ts:123]: do we want to change this? Won't things break in devnet because it's not conformed to the right structure?

**Review by Taylor123 — APPROVED** [review]
looks good. 1 minor concern about devnet data since the structure is changing, but the devnet API is not

**Comment by tomjohn1028** [comment]
>looks good. 1 minor concern about devnet data since the structure is changing, but the devnet API is not

Good call, Aleph doesn't support devnet. What's the move?

**Comment by Taylor123** [comment]
> > looks good. 1 minor concern about devnet data since the structure is changing, but the devnet API is not
> 
> Good call, Aleph doesn't support devnet. What's the move?

hmm less fun could be to wrap the places accessing the data and change the accessed structure conditionally

**Pull request merged**